android/build.gradle

```diff
@@ -53,7 +50,7 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
I wonder if we should just use `flutter.minSdkVersion` somehow? Did not seem to work in my testing though:

```
> Could not get unknown property 'flutter' for extension 'android' of type com.android.build.gradle.LibraryExtension.
```

Reply:
Seems like plugins can't do that.